web/datacatalog/datastore.py

```diff
@@ -46,17 +46,16 @@ def __init__(self):
 
 async def is_healthy():
     for datastore in implemented_datastores:
-        is_healthy = await datastore.is_healthy()
-        if not is_healthy:
```

Comment:
My IDE complains that is_healthy hides another object of the same name in the outer scope (namely the function name itself). Already fixed in commit e1d07d6ff934ba728e29e851eba2fe6b54a9bf74.